docs/compressors.md

```diff
@@ -1,63 +1,56 @@
-| zstd 1.4.4 -20          |  795030 us |    4315 us |      801329 | 22.01 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzo1y 2.10 -1           |    5719 us |    4346 us |     1448549 | 39.79 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzo1b 2.10 -1           |   12882 us |    4353 us |     1371249 | 37.66 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| zstd 1.4.4 -5           |   29924 us |    4373 us |      944741 | 25.95 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzo1f 2.10 -1           |   13097 us |    4660 us |     1430914 | 39.30 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| fastlz 0.1 -2           |   10776 us |    5020 us |     1389951 | 38.18 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| fastlz 0.1 -1           |   11455 us |    5118 us |     1462824 | 40.18 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzrw 15-Jul-1991 -1     |   11591 us |    5301 us |     1702140 | 46.75 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzrw 15-Jul-1991 -3     |   10862 us |    5307 us |     1535721 | 42.18 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzjb 2010               |   11024 us |    5538 us |     1852615 | 50.89 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzrw 15-Jul-1991 -5     |   19477 us |    6024 us |     1346634 | 36.99 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| quicklz 1.5.0 -1        |    6666 us |    6081 us |     1310173 | 35.99 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| tornado 0.6a -1         |   10191 us |    6431 us |     1463980 | 40.21 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| lzrw 15-Jul-1991 -4     |    9606 us |    6748 us |     1470050 | 40.38 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| tornado 0.6a -2         |   11920 us |    6932 us |     1251726 | 34.38 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| quicklz 1.5.0 -2        |   12054 us |    7179 us |     1187468 | 32.62 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| density 0.14.2 -3       |    9822 us |   10258 us |     1606968 | 44.14 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-| tornado 0.6a -3         |   19214 us |   11493 us |     1022778 | 28.09 | sorbet-stdlib-state-a2f6fab208ee500774c6faa9c62c228c3b21fe08|
-
-Note: LZ5 has a downsize though. It's really bad in compressing small pieces of data. <1KB data sizes normally _grow_ after compression.
+| lz4fast 1.9.3 -17       |    8368 us |    2023 us |     3555129 | 57.23 | uncompressed-gs-rle|
+| lzsse8fast 2019-04-18   |   30112 us |    2065 us |     2301226 | 37.05 | uncompressed-gs-rle|
+| lzsse4fast 2019-04-18   |   29646 us |    2189 us |     2306934 | 37.14 | uncompressed-gs-rle|
```

Comment:
I can't really make a connection between the cited settings above and the line in this table which corresponds to those settings.  Is it one of these, or a different line?

Reply:
Good question. I'm using `lz4 1.9.3` as we _do_ care a bit about compression speed _and_ size of the compressed output. These variants either have slow compression or bad compression ratio. I can point out / bold the line we are using.